src/Foundation/NSObject2.cs

```diff
@@ -72,7 +72,7 @@ public partial class NSObject
 
 		// This enum has a native counterpart in runtime.h
 		[Flags]
-		enum Flags : byte {
+		enum Flags : ushort {
```

Comment:
Any changes to this enum, especially its size, require several changes in the runtime code as well (runtime.m/runtime.h/etc).

Reply:
I suppose we can't break xamarin_set_nsobject_flags, so can I add a Flags2 enum?

Reply:
Yes, that's not a bad idea. The new enum should be internal though.